examples/tut-6-rpc/client-go/index.go

```diff
@@ -0,0 +1,214 @@
+package main
+
+import (
+  "fmt"
+  "log"
+  "os"
+  "reflect"
+  "strings"
+  "sync"
+  "time"
+  "encoding/json"
+  "io/ioutil"
+  "math/rand"
+  "net/http"
+)
+
+
+type account   = float64
+type contract  = float64
+type bigNumber = map[string]interface {}
+
+
+func dieIf(err error) {
+  if err != nil {
+    log.Fatal(err)
+    os.Exit(1)
+  }
+}
+
+
+func please(a interface{}, err error) interface{} {
```

Comment:
What's the story with this?

Reply:
It's apparently commonplace to repeat the code you see in `please` and `dieIf` everywhere it's possible to receive a failure condition (so... constantly), so I made these helpers and was able to clean things up considerably before whittling down+consolidating things to the point where they are now.

I figured you'd ask. Want me to inline them instead?

Reply:
Naw... that's nuts though.